src/app/auth/services/auth.service.ts

```diff
@@ -150,6 +160,10 @@ export const doLogin = async (
       localStorageService.set('xMnemonic', clearMnemonic);
       localStorageService.set('xNewToken', newToken);
 
+      const argon2Hash = await passToHash({ password });
+
+      authClient.upgradeHash(argon2Hash.hash);
```

Comment:
Be careful, according to the SDK [PR](https://github.com/internxt/sdk/pull/251/files) I have seen, this is a promise and you should `await` it.